pyjokes/jokes_en.py

```diff
@@ -82,26 +77,16 @@
     "The best thing about a Boolean is even if you are wrong, you are only off by a bit.",
     "A good programmer is someone who always looks both ways before crossing a one-way street.",
     "There are two ways to write error-free programs; only the third one works.",
-    "QAs consist of 55% water, 30% blood and 15% Jira tickets.",
-    "Sympathy for the Devil is really just about being nice to QAs.",
-    "How many QAs does it take to change a light bulb? They noticed that the room was dark. They don't fix problems, they find them.",
     "A programmer crashes a car at the bottom of a hill, a bystander asks what happened, he says \"No idea. Let's push it back up and try again\".",
     "What do you mean 911 is only for emergencies, I've got a merge conflict and it doesn't look good.",
-    "Writing PHP is like peeing in the swimming pool, everyone did it, but we don't need to bring it up in public.",
-    "Why did the QA cross the road? To ruin everyone's day.",
```

Comment:
This seems a little harsh without context. Perhaps "To find a pothole" or "To find a pothole and ruin someone's day" would make it work?